### DISCUSSION TITLE
change hollow tick to hollow cross when goal met but not done today

### DISCUSSION THREAD

**Discussion by DeWarner** [iSoron/uhabits#1209] [Feature Requests]
When the goal has been met by previous days the cross is not displayed as a cross, but as a hollow glowing tick.
It looks confusing when the cross is not a cross, i think an x in the same style would make more sense.

**Comment by pllama** [comment]
I also find the hollow tick confusing. I much preferred  when it was displayed as a simple gray tick ✅

**Comment by LordJon** [comment]
I'm still on the old version with the grey checkmark. Wasn't the hollow one made to be more clear that it has been fulfilled by previous days? Because you can't really see a difference between a grey cross and a grey checkmark. I'm actually looking forward to the hollow checkmark.

**Reply by DeWarner** [reply]
i dont understand, I dont remember what it was like before, but i think theres 2 parts to it:
- when you did something that day there should be a tick, if you didnt it should be a cross. I dont think that should ever be violated which is the problem with the way it is in the version i am on, 
- the blue glow seems to be the way to signal that the target has been met, and i think thats good,

my issue with it is that there is a solid blue tick for if you did it, and a hollow blue tick if you didnt. violating expectation #1

my proposed solution is to simply change the hollow blue tick to a hollow blue cross

**Comment by ke352802081770314** [comment]
I think the hollow tick is more consistent than a cross would be. A cross says that I failed the goal. The hollow tick says that my behaviour complies with the goals I have set for myself, just not by doing it that day but by doing it often enough. Fulfilling my goal should be visualized by a positive symbol, so tick is appropriate and I would find a cross confusing. How could I maintain a good score by days that have a cross? To me, that would be counter-intuitive.
I you find you are lacking something while a hollow tick is shown, then maybe you have set your goal lower than what you are planning to do. I think a hollow checkmark is the perfect visualization for "I haven't done it today but that's perfectly okay because I don't have to". It still gives me the option to tick it and make it a solid check if I feel inclined to do it anyway (and it will extend the stretch of the hollow tick).
